shared/src/summarize_result.js

```diff
@@ -63,50 +66,50 @@ async function setup() {
       }
 
       summaryResultElement.style.display = '';
-      summaryResultElement.innerHTML = data.summary.replaceAll(/\n/g, '<br />');
+      summaryResultElement.innerText = summaryTextContents;
```

Comment:
This was the other piece required to be changed for the Firefox for Android extension to not show a warning.